android/src/toga_android/widgets/selection.py

```diff
@@ -20,14 +21,90 @@ def onNothingSelected(self, parent):
         self.impl.on_change(None)
 
 
+class TogaArrayAdapter(dynamic_proxy(SpinnerAdapter)):
+    def __init__(self, impl):
+        super().__init__()
+        self.impl = impl
+        self._default_textsize = -1
+        self._default_typeface = None
+        self.adapter = ArrayAdapter(
+            self.impl._native_activity, R.layout.simple_spinner_item
+        )
+        self.adapter.setDropDownViewResource(R.layout.simple_spinner_dropdown_item)
+
+    def apply_font(self, tv):
+        if self.impl._font_impl and tv:
```

Comment:
How/when does this method get invoked when tv is None?

Reply:
During the creation of the widget, this can happen.